.github/workflows/cri_test.yml

```diff
@@ -68,3 +68,49 @@ jobs:
     - name: Cleaning
       if: ${{ always() }}
       run: ./scripts/github_runner/clean_cri_runner.sh
+  
+  gvisor-cri-tests:
+    name: gVisor CRI tests
+    env:
+        GITHUB_RUN_ID: ${{ github.run_id }}
+        GITHUB_VHIVE_ARGS: "-dbg"
+    runs-on: [self-hosted, cri]
+
+    steps:
+
+    - name: Host Info
+      run: |
+        echo $HOSTNAME
+        echo $GITHUB_RUN_ID
+
+    - name: Setup TMPDIR
+      run: mkdir -p $TMPDIR
+
+    - name: Set up Go 1.16
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1.16
+
+    - name: Check out code into the Go module directory
+      uses: actions/checkout@v2
+
+    - name: Setup gvisor-containerd
+      run: ./scripts/setup_gvisor_containerd.sh
+
+    - name: Build
+      run: go build
+
+    - name: Run vHive gVisor CRI tests
+      run: make test-cri-gvisor
+
+    - name: Archive log artifacts
+      if: ${{ always() }}
+      uses: actions/upload-artifact@v2
+      with:
+        name: ctrd-logs
+        path: |
+          /tmp/ctrd-logs/${{ github.run_id }}
+
+    - name: Cleaning
+      if: ${{ always() }}
+      run: ./scripts/github_runner/clean_cri_runner.sh
```

Comment:
you need to augment [this script](https://github.com/ease-lab/vhive/blob/58b2b6a8fa568d94f98c5da8c199855700d7897b/scripts/github_runner/clean_cri_runner.sh#L39) to carefully kill all the daemons that you spawn in the test, particularly `gvisor-containerd`. otherwise it might stay alive after the test

also, check that repeated experiments work without leaking storage. as a good start, please manually check the locations that that same script removes for firecracker

Reply:
I can see that shim also remains. please kill in the cleaning script
![image](https://user-images.githubusercontent.com/15651770/126392373-f7794cc5-fb7d-45f0-9b43-3ff75643e4d8.png)